eox_tenant/models.py

```diff
@@ -95,8 +64,8 @@ class Microsite(models.Model):
     """
     key = models.CharField(max_length=63, db_index=True)
     subdomain = models.CharField(max_length=127, db_index=True)
-    values = JSONField(null=False, blank=True)
-    objects = MicrositeManager()
+    values = JSONField(blank=True, default={}, load_kwargs={'object_pairs_hook': collections.OrderedDict})
```

Comment:
Does this change in any way imply that we need to change the data in the database?